openedx/core/djangoapps/notifications/views.py

```diff
@@ -51,7 +52,12 @@ class CourseEnrollmentListView(generics.ListAPIView):
 
     def get_queryset(self):
         user = self.request.user
-        return CourseEnrollment.objects.filter(user=user, is_active=True)
+        enrollments = CourseEnrollment.objects.filter(user=user, is_active=True)
+        return [
+            enrollment
+            for enrollment in enrollments
+            if ENABLE_NOTIFICATIONS.is_enabled(enrollment.course.id)
```

Comment:
is it possible to fetch all of the courses that have this flag enabled?

Reply:
I don't think it is possible to get all courses with this flag enabled